docs/en/commands/README.md

```diff
@@ -0,0 +1,155 @@
+## Appium API Documentation
```

Comment:
is there any way for the auto-generated files to have a comment at the top saying do not edit?

Reply:
Ah yes, good idea.